interface/backend/api/tests.py

```diff
@@ -29,20 +30,23 @@ def test_nodule_list_viewset(self):
         payload = response.json()
         self.assertListEqual(payload, [])
 
-        # now create a nodule and figure out what we expect to see in the list
+        # now create nodules and figure out what we expect to see in the list
         case = CaseFactory()
         candidates = CandidateFactory.create_batch(size=3, case=case)
         nodules = []
         for candidate in candidates:
             nodule, _ = candidate.get_or_create_nodule()
             nodules.append(nodule)
         serialized = [NoduleSerializer(n, context={'request': None}) for n in nodules]
-        expected = [s.data for s in serialized]
+        expecteds = [json.loads(json.dumps(s.data)) for s in serialized]
```

Comment:
Hang on, what? :)

Reply:
This but `.data` of a serialized instance is an `OrderedDict` with non-JSON-primitive objects (e.g. an `enum.Enum` instance rather than `1`), so for comparing values with `assertDictEqual` it was easier to simulate a trip 'round the API (serialized to JSON, then unserialized).

I agree that this is gross but not sure how else to deal with it other than avoiding `assertDictEqual`. Any suggestions would be welcome.

Reply:
Perhaps add a comment to that effect to remind future you? :)

Reply:
Mmm,. please add as a comment :)